site/src/analytics-integrations.ts

```diff
@@ -205,6 +218,7 @@ export class AnalyticsIntegrations extends LitElement {
   private _clearFilter() {
     this._currentTablePage = 0;
     this._filter = "";
+    window.location.search = "";
```

Comment:
You should do this after you consumed the filter from the URL. Not every time.

Reply:
I'll move all handling to the main element

Reply:
Added a guard